images/users/disqus.svg

```diff
@@ -0,0 +1 @@
+<svg width="438" height="175" viewBox="0 0 438 175" xmlns="http://www.w3.org/2000/svg"><title>Slice</title><path d="M30.208 49.602H.878v76.04h28.895c27.376 0 43.342-16.294 43.342-38.237v-.22c.002-21.94-15.75-37.583-42.907-37.583zm21.29 38.128c0 12.272-8.363 19.227-20.963 19.227h-8.582v-38.67h8.582c12.6 0 20.964 7.06 20.964 19.225v.218zm35.63 37.907h21.182V49.602H87.128v76.035zm70.065-47.358c-10.538-2.392-13.145-3.912-13.145-7.386v-.22c0-2.716 2.5-4.888 7.604-4.888 6.733 0 14.338 2.605 21.292 7.496l10.536-14.885c-8.257-6.625-18.358-10.103-31.284-10.103-18.25 0-29.438 10.21-29.438 24.332v.218c0 15.644 12.492 20.207 28.676 23.896 10.32 2.39 12.818 4.127 12.818 7.28v.216c0 3.258-3.042 5.213-8.69 5.213-8.798 0-17.163-3.15-24.657-9.124l-11.732 14.014c9.342 8.365 22.16 12.604 35.63 12.604 18.466 0 30.742-9.233 30.742-24.768v-.217c0-14.23-10.865-19.878-28.352-23.68zm120.573 9.342v-.218c0-21.834-17.38-39.324-40.844-39.324-23.462 0-41.06 17.708-41.06 39.542v.216c0 21.834 17.38 39.324 40.845 39.324 8.69 0 16.62-2.5 23.137-6.736l8.363 7.494 10.645-11.84-7.82-6.623c4.343-6.193 6.734-13.8 6.734-21.835zm-21.398.433c0 2.607-.435 5.105-1.304 7.385l-10.32-9.342-10.644 11.95 10.537 9.017c-2.28.87-4.887 1.412-7.71 1.412-11.622 0-19.444-9.666-19.444-20.64v-.215c0-10.972 7.712-20.532 19.225-20.532 11.733 0 19.664 9.668 19.664 20.748v.216h-.004zm83.534 4.345c0 10.643-5.543 15.64-14.016 15.64-8.47 0-14.012-5.212-14.012-16.185V49.602h-21.398v42.69c0 23.79 13.578 34.763 35.195 34.763 21.618 0 35.63-10.754 35.63-35.305V49.602h-21.398V92.4zm68.868-14.12c-10.537-2.392-13.145-3.912-13.145-7.386v-.22c0-2.716 2.5-4.888 7.607-4.888 6.73 0 14.338 2.605 21.29 7.496L435.058 58.4c-8.255-6.625-18.356-10.103-31.282-10.103-18.25 0-29.44 10.21-29.44 24.332v.217c0 15.644 12.493 20.205 28.677 23.896 10.32 2.39 12.818 4.13 12.818 7.28v.216c0 3.257-3.043 5.214-8.69 5.214-8.8 0-17.165-3.152-24.66-9.125l-11.728 14.01c9.343 8.368 22.16 12.603 35.627 12.603 18.468 0 30.742-9.233 30.742-24.766v-.22c-.002-14.227-10.866-19.875-28.354-23.678z" fill="#229CFF" fill-rule="evenodd"/></svg>
```

Comment:
Thanks for your PR. Could you resize your SVG to 500x200 as mentionned at the bottom of the user page.